src/plugins/timelion/public/application.ts

```diff
@@ -1,129 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0 and the Server Side Public License, v 1; you may not use this file except
- * in compliance with, at your election, the Elastic License 2.0 or the Server
- * Side Public License, v 1.
- */
-
-import './index.scss';
-
-import { EuiIcon } from '@elastic/eui';
-import angular, { IModule } from 'angular';
-// required for `ngSanitize` angular module
-import 'angular-sanitize';
-// required for ngRoute
-import 'angular-route';
-import 'angular-sortable-view';
```

Comment:
As far as I can see, this package isn't used anywhere else in Kibana and can be removed from the `package.json` file

Reply:
@watson  'angular-sanitize' and 'angular-route' are using in `x-pack\plugins\monitoring\public\angular\app_modules.ts`. We cannot remove them for now. I can remove only 'angular-sortable-view'.

Reply:
Sorry, yean I meant just `angular-sortable-view`. I guess the diff above just shows a bit more than what I'm commenting on 😅

Reply:
Okay, I've removed `angular-sortable-view`